test/Interop/Cxx/class/loadable-types-silgen.swift

```diff
@@ -0,0 +1,116 @@
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithInheritedMoveConstructor)
+func pass(s: StructWithInheritedMoveConstructor) {
+  // CHECK: bb0(%0 : $*StructWithInheritedMoveConstructor):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithSubobjectMoveConstructor)
+func pass(s: StructWithSubobjectMoveConstructor) {
+  // CHECK: bb0(%0 : $*StructWithSubobjectMoveConstructor):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithCopyAssignment)
+func pass(s: StructWithCopyAssignment) {
+  // CHECK: bb0(%0 : $*StructWithCopyAssignment):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithInheritedCopyAssignment)
+func pass(s: StructWithInheritedCopyAssignment) {
+  // CHECK: bb0(%0 : $*StructWithInheritedCopyAssignment):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithSubobjectCopyAssignment)
+func pass(s: StructWithSubobjectCopyAssignment) {
+  // CHECK: bb0(%0 : $*StructWithSubobjectCopyAssignment):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithMoveAssignment)
+func pass(s: StructWithMoveAssignment) {
+  // CHECK: bb0(%0 : $*StructWithMoveAssignment):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithInheritedMoveAssignment)
+func pass(s: StructWithInheritedMoveAssignment) {
+  // CHECK: bb0(%0 : $*StructWithInheritedMoveAssignment):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithSubobjectMoveAssignment)
+func pass(s: StructWithSubobjectMoveAssignment) {
+  // CHECK: bb0(%0 : $*StructWithSubobjectMoveAssignment):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithDestructor)
+func pass(s: StructWithDestructor) {
+  // CHECK: bb0(%0 : $*StructWithDestructor):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithInheritedDestructor)
+func pass(s: StructWithInheritedDestructor) {
+  // CHECK: bb0(%0 : $*StructWithInheritedDestructor):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithSubobjectDestructor)
+func pass(s: StructWithSubobjectDestructor) {
+  // CHECK: bb0(%0 : $*StructWithSubobjectDestructor):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructTriviallyCopyableMovable)
+func pass(s: StructTriviallyCopyableMovable) {
+  // CHECK: bb0(%0 : $StructTriviallyCopyableMovable):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructNonCopyableTriviallyMovable)
+func pass(s: StructNonCopyableTriviallyMovable) {
+  // CHECK: bb0(%0 : $StructNonCopyableTriviallyMovable):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructNonCopyableNonMovable)
+func pass(s: StructNonCopyableNonMovable) {
+  // CHECK: bb0(%0 : $StructNonCopyableNonMovable):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructDeletedDestructor)
+func pass(s: StructDeletedDestructor) {
+  // CHECK: bb0(%0 : $StructDeletedDestructor):
+}
```

Comment:
Shouldn't the last three cases be address-only?

Reply:
Yes. I just wasn't done, yet :-)